blog/2024-11-22-terraform-repo-move.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Terraform Component GitHub Repository Has Moved!
+authors: [goruha]
+slug: terraform-components-have-moved
+tags: [terraform, github, migration]
+date: 2024-11-22
+---
+
+import Intro from '@site/src/components/Intro';
+import Steps from '@site/src/components/Steps';
+
+<Intro>
+The GitHub repository for Cloud Posse's Terraform components has migrated to a dedicated GitHub organization. All documentation remains here, but all future updates, contributions, and issue tracking for the source code should now be directed to the respective repositories in the new organization.
+</Intro>
+
+# Hello, Cloud Posse Community!
+
+We're excited to announce that starting on November 12, 2024, we will begin migrating each component in the `cloudposse/terraform-aws-components` repository to individual repositories under a new GitHub organization. This change aims to improve the stability, maintainability, and usability of our components.
```

Comment:
Here the date is 11/12, but the post is 11-22. Maybe that's an oversight as it doesn't make sense that we announced after we started that we're starting.

Reply:
changed both to 11-12